bin/rialto/node/src/chain_spec.rs

```diff
@@ -191,3 +193,30 @@ fn load_kovan_bridge_config() -> BridgeKovanConfig {
 		initial_validators: rialto_runtime::kovan::genesis_validators(),
 	}
 }
+
+// We use this to get the account on Rialto (target) which is derived from Millau's (source) Dave
+// account. We do this so we can fund the derived account on Rialto at Genesis to it can pay
+// transaction fees. The reason we can use Rialto's Dave for the `millau_account_id` is because
+// they're both derived from the same seed phrase.
+//
+// In case you need it, the derived account ID is: 5Hg7WQyk8C1FmPzxY3xSjR7S6zZZC5sAL35vMr6NpW17jBhQ
```

Comment:
IMHO it makes more sense to add this as a test case, so that whenever it changes we will clearly see it ;)